azurerm/internal/services/policy/policy_assignment_resource.go

```diff
@@ -129,7 +135,7 @@ func resourceArmPolicyAssignmentCreateUpdate(d *schema.ResourceData, meta interf
 
 	name := d.Get("name").(string)
 	scope := d.Get("scope").(string)
-
+	enforcementMode := convertEnforcementMode(utils.Bool(d.Get("enforcement_mode").(bool)))
```

Comment:
It doesn't look like we need a pointer to a boolean so we should remove `utils.Bool` 
```suggestion
	enforcementMode := convertEnforcementMode(d.Get("enforcement_mode").(bool))
```

Reply:
thanks thats a left over from a previous testing to find a problem. I removed it see latest commit